lib/ioki/apis/endpoints/endpoints.rb

```diff
@@ -34,10 +38,19 @@ def self.crud_endpoints(resource, model_class:, base_path:, paths: {}, except: n
 
         type_key = type.to_s.split('::').last.downcase.to_sym
         name = [Index].include?(type) ? plural : singular
+        action_name = type.to_s.split('::').last.downcase.to_sym
```

Comment:
What's the difference between the variables `type_key` and `action_name`? They're both assigned with the same expression.

Reply:
Wow, I really didn't see that :see_no_evil: Fixed!